airbyte-commons-worker/src/main/java/io/airbyte/workers/process/AsyncOrchestratorPodProcess.java

```diff
@@ -298,6 +301,33 @@ public void create(final Map<String, String> allLabels,
     final List<ContainerPort> containerPorts = KubePodProcess.createContainerPortList(portMap);
     containerPorts.add(new ContainerPort(serverPort, null, null, null, null));
 
+    final var initContainer = new ContainerBuilder()
+        .withName(KubePodProcess.INIT_CONTAINER_NAME)
+        .withImage("busybox:1.35")
+        .withVolumeMounts(volumeMounts)
+        .withCommand(List.of(
+            "sh",
+            "-c",
+            String.format("""
```

Comment:
Nit:  It may make sense to extract this script to a classpath resource just to shorten this method/make it easier to read.  The contents of the file can still be fed to the format string.